gwt-material/src/test/java/gwt/material/design/client/ui/MaterialModalTest.java

```diff
@@ -41,21 +41,6 @@ public void init() {
         checkType(modal);
         checkDimissible(modal);
         checkDuration(modal);
-        checkMultipleModalZIndexes();
-    }
-
-    protected void checkMultipleModalZIndexes() {
```

Comment:
I have a basic test case that check Z indexes in many cases that previously were problematic:

```
    protected void checkMultipleModalZIndexes() {
    	// Open several modals and check Z indexes
    	List<MaterialModal> modals = new ArrayList<>();
        for (int i = 0; i <= 5; i++) {
            modals.add(createModal());
        }
        checkMultipleModalZIndexes(modals);
        // Close last modal and check again
        modals.remove(modals.size() - 1).close();
        checkMultipleModalZIndexes(modals);
        // Close first modal and check again
        modals.remove(0).close();
        checkMultipleModalZIndexes(modals);
        // Create new modal and check again
        modals.add(createModal());
        checkMultipleModalZIndexes(modals);
        // Create one more new modal and check again
        modals.add(createModal());
        checkMultipleModalZIndexes(modals);
    }
    
    protected MaterialModal createModal() {
        MaterialModal modal = new MaterialModal();
        RootPanel.get().add(modal);
        modal.open();
        return modal;
    }
    
    protected void checkMultipleModalZIndexes(List<MaterialModal> modals) {
    	MaterialModal previousModal = null;
    	for (MaterialModal modal : modals) {
            if (previousModal != null) {
            	// Check that this modal is above previous one
                assertTrue(Integer.valueOf(modal.getElement().getStyle().getZIndex()) > Integer.valueOf(previousModal.getElement().getStyle().getZIndex()));
                // TODO Check if overlay from this modal covers previousModal?
            }
            previousModal = modal;
    	}
    }
```

However, real deal is in given TODO, which I'm unable to solve due to my lack of experience with jQuery and JSInterop: how to test if current dialog's overlay covers all previous modals? In materializejs every modal keeps reference to its overlay's ID:

```
$modal.data('overlay-id', overlayID)
```

but I'm not sure how to fetch that through JS Interop.